contents/docs/libraries/js/index.mdx

```diff
@@ -342,6 +342,74 @@ if (posthog.isFeatureEnabled('new-groups-feature')) {
 
 To check flag status for a different group, first switch the active group by calling `posthog.group()`.
 
+## Surveys
+
+[Surveys](/docs/surveys) launched with [popover presentation](/docs/surveys/creating-surveys#display-mode) are automatically shown to users matching the targeting you set up.
```

Comment:
hyperlink "targeting" to the survey docs section, so it's clear what exactly we mean by it